src/views/home.hbs

```diff
@@ -1,4 +1,28 @@
-<div> 
-    Hello from home page
+<div id="proposal">
+    <div>
+        Create new post:
+    </div>
+    <textarea onfocus="append()" onblur="remove()" id="text-area" placeholder="Write your proposal..."></textarea>
```

Comment:
@Anies12 I can't remember if there is a max length for the description field in your database - if there is, you could set the same `maxlength=` attribute on this element too.